src/main/paradox/kubernetes-deployment.md

```diff
@@ -7,9 +7,47 @@ Once you've installed the deployment tool, `reactive-cli`, you can use the `rp`
 * [reactive-cli](https://github.com/lightbend/reactive-cli) installed
 * Applications built and published to a registry (production) or the local docker engine ([Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/))
 * `kubectl` installed and configured to point to your registry
+*  Kubernetes cluster with correctly configured Role-Based Access Control ([RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)). 
 
 > Using Minikube? Make sure you run `eval $(minikube docker-env)` before following the steps below. When you need your old environment, you can get it back with `eval $(minikube docker-env -u)`.
 
+### RBAC
+
+Many production clusters and recent versions of Minikube come with authorization feature called [Role-Based Access Control](https://kubernetes.io/docs/admin/authorization/rbac/). Basic idea is to give users certain permissions when working with the cluster. Sets of these permissions are called roles. Users are given roles using role bindings. Note that even a process running inside a pod belong to a special type of user called [Service Account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). 
+
```

Comment:
pedantic note:  let's say "service accounts" instead of "users" here.  The key here is that Kubernetes doesn't have a user object--users are managed by external authentication mechanisms which we don't need in this case.  Here we're referencing permissions meant for service to service permissions, rather than person to service permission.  Think of this as the permissions the app needs to talk to the api server, which should be granted via a service account.

Reply:
more on "users":  https://github.com/lightbend/k8s-explore/tree/master/users

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/rbac/ says:

> Role-based access control (RBAC) is a method of regulating access to computer or network resources based on the roles of individual users within an enterprise.
>
> A role binding grants the permissions defined in a role to a user or set of users. 

See also https://kubernetes.io/docs/concepts/policy/pod-security-policy/

> Most Kubernetes pods are not created directly by users. Instead, they are typically created indirectly as part of a Deployment, ReplicaSet, or other templated controller via the controller manager.

So Kubernetes might not let you treat users as a resource, but I think it does have the notion of human users as well as non-human users.